예산.js

```diff
@@ -0,0 +1,14 @@
+function solution (d, budget) {
+    let answer = 0, sum = 0;
+    d.sort ((a,b) => a - b);
+
+    for (let i = 0; i < d.length; i++){
+        answer++;
+        sum += d[i]
+
+        if (sum > budget)
+            answer--;
+    }
```

Comment:
1. 문단이 이어지기 때문에 줄바꿈은 놉!
2. 실행문이 한 줄이라서 {}를 생략하신거 같은데, 생략 할때는 무조건 한줄 입력~
```suggestion
        if (sum > budget) answer--;
    }
```